docs/usage/types.md

```diff
@@ -88,9 +88,19 @@ with custom properties and validation.
 `typing.Tuple`
 : see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
 
+`subclass of typing.NamedTuple (or collections.namedtuple)`
+: Same as `tuple` but instantiates with the given namedtuple.  
+ _pydantic_ will validate the tuple if you use `typing.NamedTuple` since fields are annotated.  
+  If you use `collections.namedtuple`, no validation will be done.  
```

Comment:
I think "no validation will be done" is misleading. I assumed it meant not validation at all will be done, e.g. the field would be equivalent to `Any`.

I think better to make `collections.namedtuple` a different item in this list and instead say "all fields will have type `Any`" or something.